src/rdkafka_cgrp.c

```diff
@@ -697,18 +861,24 @@ rd_kafka_rebalance_op (rd_kafka_cgrp_t *rkcg,
 		       const char *reason) {
 	rd_kafka_op_t *rko;
 
+        if (err == RD_KAFKA_RESP_ERR__REVOKE_PARTITIONS)
+                rkcg->rkcg_flags |= RD_KAFKA_CGRP_F_WAIT_UNASSIGN_CALL;
```

Comment:
I recall moving this in here having reasoned I was not changing existing logic in doing so. However, i don't see that now when investigating.

For the moment, leaving your question as a TODO, will get back to this.

Reply:
you're right, i did screw this up. moving this effectively back to how it was.

Reply:
My new logic is in fact correct (removed my comment just now that errantly says it's not!). I had moved this in here because i thought it better belonged there, but i've changed my mind on that and moving it back to how you had it.